### ISSUE TITLE
Encountered difficulties in creating the project and cannot run (pre-built Maven JAR is not working and report FileNotFound)

### ISSUE THREAD

**Issue by leck995** [brianhsu/Live2DForScala#2]
Hi, I am not very good at scala and sbt, so I tried to use this library through Java, but I encountered difficulties. 
  I created a new project through Maven according to the tutorial, imported the core into the project, and ran the following code, but encountered an error. I have also run it using scala, and the error is consistent.
  Then I tried running the same code using Java in the example directory of your project, and it ran successfully. I am not sure what was overlooked, so I do not know how to solve the problem. 
I need help，plaese.

![error](https://github.com/brianhsu/Live2DForScala/assets/46077555/b38c6b48-fe33-4e70-909b-dbb318c32ef2)

**Comment by leck995** [comment]
翻了您的倉庫，發現有中文，故重發壹下中文。

妳好，因爲我不了解scala與sbt，故我嘗試通過java來使用這個庫，但是我遇到了困難。
我按照教程通過maven創建了新項目，同時將core導入項目，然後運行如下代碼，産生了錯誤，我以爲是由于java導致的，故重新用maven下和sbt下的scala再壹次運行，錯誤都是壹致。
然後我嘗試在您所創建的的項目example目錄下使用java運行相同的代碼，成功運行。我不清楚疏漏了什麽，故而不知道如何解決該問題，請幫助我。謝謝

**Comment by brianhsu** [comment]
Thanks for your reporting, I've confirm this is a bug in the pre-built maven library.

I will look into it when I'm available. The `example` works because it does not use the pre-built library.

If you really to use this library right now, I'm afraid that you need to build this library yourself and include it in you project. (Run `core/assembly` to build the fat jar for core module)

For example:
```
 Live2DForScala $ sbt
 sbt:live2dforscala> core/assembly
sbt:live2dforscala> core/assembly
[info] Strategy 'discard' was applied to 9 files (Run the task at debug level to see details)
[info] Strategy 'rename' was applied to a file (Run the task at debug level to see details)
[info] Assembly up to date: /mnt/Data/WorkRoom/Live2DForScala/modules/core/target/scala-2.13/Core-assembly-2.0.1-SNAPSHOT.jar
```

Then copy the Core-assembly-2.0.1-SNAPSHOT.jar into your maven project and assign it as a dependency in your IDE. It should work.

Same procedure if you need OpenGL bindings.

**Comment by brianhsu** [comment]
It should be fixed in v2.0.1. 
Please try update your pom.xml to use the latest version of this library and the issue should be gone, thanks.

**Comment by leck995** [comment]
Thank you, the problem has been successfully resolved. This is a fantastic project that surprised me very much.
